includes/Admin/Admin_AJAX.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * Class WordPress\Plugin_Check\Admin\Admin_AJAX
+ *
+ * @package plugin-check
+ */
+
+namespace WordPress\Plugin_Check\Admin;
+
+/**
+ * Class is handling admin tools page functionality.
+ *
+ * @since n.e.x.t
+ */
+class Admin_AJAX {
+
+	/**
+	 * Nonce key.
+	 *
+	 * @since n.e.x.t
+	 * @var string
+	 */
+	private $nonce_key;
+
+	/**
+	 * Constructor.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param string $nonce_key Nonce key.
+	 */
+	public function __construct( $nonce_key ) {
+		$this->nonce_key = $nonce_key;
+	}
+
+	/**
+	 * Initializes hooks.
+	 *
+	 * @since n.e.x.t
+	 */
+	public function add_hooks() {
+		add_action( 'wp_ajax_plugin_check_run_check', array( $this, 'run_check' ) );
```

Comment:
Based on discussions with @felixarntz in #84 this should be renamed to `wp_ajax_plugin_check_run_checks` and the method here to `run_checks`